api/models.py

```diff
@@ -1,3 +1,21 @@
 from django.db import models
-
+from django.utils import timezone
 # Create your models here.
+
+
+class Projects(models.Model):
+	PROJECT_STATUS = (
+		('O', 'Opened'),
+		('I', 'In progress'),
+		('C', 'Closed'),
+	)
+	project_name = models.CharField(max_length=50)
+	project_description = models.CharField(max_length=150, blank=True)
+	status_id = models.CharField(max_length=1, choices=PROJECT_STATUS, blank=True)
+	project_deadline = models.DateTimeField(blank=True)
+	repository_link = models.URLField(max_length=200, blank=True)
+	website_prod = models.URLField(max_length=200, blank=True)
```

Comment:
_prod? Just use full word. Screens are wide and explicit is easier to remember/read.